### PR TITLE
Drop erc4337 userops legacy

### PR DESCRIPTION
**Pull request by aalan3**
This is a good first test because it takes some time to run and also has a clean lineage.

### CODE REVIEW

**Comment by aalan3** [comment]
I reverted the full drop for now because validation requires us to have a dummy model for the schema.

File: models/account_abstraction/erc4337/arbitrum/account_abstraction_erc4337_arbitrum_userops_legacy.sql [hunk 1]
```diff
@@ -1,99 +1,9 @@
 {{ config(
 	tags=['legacy'],
-	
-    alias = alias('userops', legacy_model=True),
```
[the remark this answers is not in the document]
Comment (jeff-dude) [reply models/account_abstraction/erc4337/arbitrum/account_abstraction_erc4337_arbitrum_userops_legacy.sql:4]: it worked on `dbt-databricks`, so i'd imagine it'll work on trino adaptor too. i'd say we give it a shot and check the before/after effect -- can look to query before/after to see results

**Comment by jeff-dude** [comment]
> I reverted the full drop for now because validation requires us to have a dummy model for the schema.

i think we should put together a quick plan on how we start this process. we could make this process simple potentially, especially at scale

one idea:
- move the models to a new directory -- `models/legacy_deprecated/` or naming similar to that -- then in prod orchestration, exclude that directory from running
- likely only exclude in incremental steps, also full table / labels runs, but leave as-is on `state:modified` so initial run happens to build schema on spark when necessary until full deprecation
- this would allow us to leave models as-is and avoid rewriting to the dummy model. the dummy models can still be built in new dunesql spells, but then moved to this new directory and only run in `state:modified` initial run
- once ready to full shut off, we can delete the whole directory and/or archive to another repo or something

**Comment by aalan3** [comment]
Closing in favour of https://github.com/duneanalytics/spellbook/pull/4188 and https://github.com/duneanalytics/spellbook/pull/4190

**Pull request closed without merging**